### DISCUSSION TITLE
Honestly, I still think you should fork Qt 5.6.3 instead

### DISCUSSION THREAD

**Discussion by ghost** [RolandHughes/copperspice#18] [General]
The Qt company will not even care if you name your fork LsQt. The Trinity Desktop Environment named their Qt3 fork as TQt3, and so far the Qt company has done nothing to them. You will tell me to do my own fork of Qt instead. But I still think I need to tell you that you should really reconsider your decision to fork CopperSpice.

p/s: If you don't need the LGPLv21 license, you should go with the latest LGPLv3 Qt5 instead of Qt 5.6.3.

**Comment by RolandHughes** [comment]
You can think what you want.

You can create your own fork and do what you want.

Despite CopperSpice claims of removing QML, you will still find a bunch of QML/Declarative source files in the Webkit directory tree. Why? Because C++ is type safe and JavaScript is not.

Forking anything beyond what they forked requires a _**massive**_ effort to purge the QML. By that time it was sooooo deeply rooted one could almost call it a lethal infection.

There are really only two things CopperSpice did correctly. 

1. Removal of MOC
2. Removal (mostly) of QML.

Keep in mind this library is focused on embedded systems development for the FDA regulated world.

I know exactly where I'm headed. If that isn't where you wish to go, please feel free to create your own.
